core/src/edu/unibo/martyadventure/view/character/EnemyCharacterView.java

```diff
@@ -1,5 +1,29 @@
 package edu.unibo.martyadventure.view.character;
 
-public class EnemyCharacterView {
+import java.util.concurrent.ExecutionException;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.math.Vector2;
+
+import edu.unibo.martyadventure.model.character.EnemyCharacter;
+
+public class EnemyCharacterView extends CharacterView {
+    
+    private static EnemyCharacter player = new EnemyCharacter(null, "Biff", 800, null );
```

Comment:
Assolutamente no, non mettere dati statici che devono essere condivisi in lettura da tutte le istanze, specialmente se puoi instanziare EnemyCharacterView liberamente come una new.